### ISSUE TITLE
FEM Subject Toolbar should have tooltip labels on hover

### ISSUE THREAD

**Issue by seanmiller26** [zooniverse/front-end-monorepo#4483]
## Is your feature request related to a problem? Please describe.
On hover a label should appear to tell volunteers what each tool does. Similar to the functionality in PFE.

<img width="463" alt="Screen Shot 2023-04-18 at 12 40 10 PM" src="https://user-images.githubusercontent.com/89043489/232860089-82571786-a566-4301-a8b6-acadc992b709.png">

## Describe the solution you'd like

- Edit tool: Annotate
- Pan tool: Pan
- Zoom tools: Zoom In, Zoom Out
- Rotate tool: Rotate
- Reset tool: Reset View
- Invert tool: Invert Colors

**Comment by eatyourgreens** [comment]
>If you want to hide content from mobile and tablet users as well as assistive tech users and keyboard only users, use the title attribute.

https://www.tpgi.com/using-the-html-title-attribute-updated/

**Comment by goplayoutside3** [comment]
Linked this related Issue that talks about possible helper labels for the toolbar: https://github.com/zooniverse/front-end-monorepo/issues/76
